_po/ja/how-to/laravel.po

```diff
@@ -103,38 +134,55 @@ msgid ""
 "DB_PASSWORD=postgres\n"
 "```"
 msgstr ""
+"```ini\n"
+"DB_CONNECTION=pgsql\n"
+"DB_HOST=127.0.0.1\n"
+"DB_PORT=5432\n"
+"DB_DATABASE=pgroonga_laravel\n"
+"DB_USERNAME=postgres\n"
+"DB_PASSWORD=postgres\n"
+"```"
 
 msgid ""
 "If you’re using different PostgreSQL username and password, change them accord"
 "ing to your settings."
 msgstr ""
+"実際に使っているPostgreSQLのユーザ名とパスワードに変更願います。"
 
 msgid "### Creating a Blog"
-msgstr ""
+msgstr "### ブログの作成"
 
 msgid ""
 "You’re now ready to start building your new application! In this tutorial, we "
 "will create a blog with fake data seeder and search through them using `PGroon"
 "ga`."
 msgstr ""
+"これで新しいアプリを作る準備ができました！このチュートリアルではブログアプリを作ります。"
+"サンプルデータを流し込む機能と`PGroonga`を使った検索機能を実装します。"
 
 msgid "### Models, migrations, and controllers"
-msgstr ""
+msgstr "### モデルとマイグレーションとコントローラの作成"
 
 msgid ""
 "To make a blog, we will need to create a model, migrations, and controllers.\n"
 "Since the blog has posts, we create Post model for it. To create a model, just"
 " run the following command:"
 msgstr ""
+"ブログを作るためのモデルとマイグレーションとコントローラを作成します。\n"
+"ブログは記事が必要なのでPostモデルを作ります。モデルを作るためには次の"
+"コマンドを実行します:"
 
 msgid ""
 "```shell\n"
 "php artisan make:model -mc Post\n"
 "```"
 msgstr ""
+"```shell\n"
+"php artisan make:model -mc Post\n"
+"```"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.